.circleci/config.yml

```diff
@@ -246,7 +246,7 @@ jobs:
             snyk test --org=${SNYK_ORG} --severity-threshold=$THRESHOLD
   link-validation:
     docker:
-      - image: cypress-13.13.0-node-20.15.1-chrome-126.0.6478.114-1-ff-128.0-edge-126.0.2592.61-1
+      - image: https://app.circleci.com/pipelines/github/usagov/vote-gov-drupal/95906/workflows/98f118ca-8bfe-4a55-8315-c033796657ec/jobs/103441
```

Comment:
This URI is not for an image.

Reply:
not sure what happened there but its fixed now